README.md

```diff
@@ -62,48 +63,46 @@ A user-focused addin for searching, creating, and editing military symbols in Ar
+		* Included in the repository releases: https://github.com/Esri/military-symbol-editor-addin-wpf/releases
+        * Install the add-in by double clicking on the downloaded .addin file
+    * Run the add-in
+		* Run ArcGIS Pro
 		* The ADD-IN appears under the "ADD-IN" tab in Pro	
 		* Click the "Military Symbol Designer" button and the tool will appear
+		* Note: If you do not already have the Military Overlay Information Model/GDB included in your project you will be prompted to add when you use the addin.
+			* Click "Yes" at the prompt: "Would you like to add the Military Overlay Layer Package to add the schema to your project?"
 		* The Search tab is the first tab:
 			* Type a search term into the bar, and click Search (or hit enter)
 			* The tool will return matches to that term in the Military style file
 			* Selecting a search result will show a preview and associated tags with that symbol
-			* Click the next arrow button or the modify tab when you have selected a style
+			* Click the next arrow button or the Modify tab when you have selected a symbol
 		* The Modify tab is the second tab:
 			* Use the select tool in ArcGIS Pro to select already existing features
 			* When selected, this will populate a list on this tab
 			* Select a feature in this list to start editing it.  The feature's data will be loading into the other tabs for editing.
 		* The Favorites tab is the third tab:
-			* Users can save a symbol that they've created so that they can easily go back to it against
+			* Users can save a symbol to Favorites that they have previously created so they can easily reuse this symbol
 			* The favorites save in a user's account, and show up on this tab
 			* Selecting a favorite from the list will load it into the pre-existing workflow.
 		* The Symbol tab is the fourth tab:
-			* The application will display all attributes associated with the chosen style, with combo boxes for selecting values
+			* The application will show all symbol display attributes associated with the chosen symbol, with combo boxes for selecting values
 			* As you select values, the symbol will update to incorporate those changes
 			* A table below has a summary of the values you select
 			* After you are finished loading values, click the next arrow or the Text tab
 		* The Label tab is the fifth tab:
-			* Various values that will be saved into the feature are available for edit here.  These values will appear around the symbol on the map.
+			* Various values that will be saved into the feature are available for edit here. These values will appear around the symbol on the map.
 			* After you are finished populating any text values, you can click the next arrow button to go the last tab.
-		* The Coordinate tab is the sixth tab:
+		* The Enter Coordinates tab is the sixth tab:
 			* Coordinates can be entered in DD, DMS, DD, or other coordinate systems.
```

Comment:
Sure -or- we could just like to the library for the full list (in case it changes?): https://github.com/Esri/coordinate-conversion-addin-dotnet#features